source/blog/individual.html.erb

```diff
@@ -9,6 +9,7 @@ include_campaign_monitor: true
   locale_obj = locals[:locale_obj] || current_locale_obj
   has_sharesight20_category = sharesight20_category && post[:categories]&.any?{ |category| category[:id] == sharesight20_category[:id] }
   show_menu = !has_sharesight20_category
+  first_category = post[:categories]&.first
```

Comment:
there should be a fallback in case this is nil - posts are not required to have a category assigned (although I think they always do in practice)